src/docs/stan-reference/distributions.tex

```diff
@@ -1641,27 +1641,43 @@ \subsubsection{Stan Functions}
  Rayleigh cumulative distribution of \farg{y} given scale \farg{sigma}}
 %
 \fitem{real}{rayleigh\_ccdf\_log}{real \farg{y}, real
-\farg{sigma}}{The log of the Rayleigh complementary cumulative distribution 
+\farg{sigma}}{The log of the Rayleigh complementary cumulative distribution
  of \farg{y} given scale \farg{sigma}}
 \end{description}
 %
 \begin{description}
 \fitem{real}{rayleigh\_rng}{real
   \farg{sigma}}{Generate a Rayleigh variate with scale
-  \farg{sigma}; may only be used in generated 
+  \farg{sigma}; may only be used in generated
 quantities block}
 \end{description}
 
 \section{Wiener Diffusion Model Distributions}
 
+\subsubsection{Probability Density Function}
+ If $\alpha \in \posreals$, $\tau \in \posreals$,
+    $\beta \in [0, 1]$ and $\delta \in \reals$,
+    then for $y \in [0,\infty)$,
+\[
+\distro{Wiener}(y|\alpha, \tau, \beta, \delta)
+=
+\frac{\alpha}{(y-\tau)^3/2}
+\exp \left(- \delta \alpha \beta - \frac{\delta^2(y-\tau)}{2}\right)
+\sum_{k = - \infty}^{\infty} (2k + \beta) \varphi \left(\frac{2k + \alpha \beta}{\sqrt{y - \tau}}\right)
+\!.
+\]
+
+where $\varphi(x)$ denotes the standard
```

Comment:
thanks. I'll just switch it to $\phi$ and we'll get it in.

On Sun, Aug 2, 2015 at 4:10 PM, Marco Inacio <notifications@github.com>
wrote:

> In src/docs/stan-reference/distributions.tex
> <https://github.com/stan-dev/stan/pull/1578#discussion_r36047123>:
>
> >  quantities block}
> >  \end{description}
> >
> >  \section{Wiener Diffusion Model Distributions}
> >
> > +\subsubsection{Probability Density Function}
> > + If $\alpha \in \posreals$, $\tau \in \posreals$,
> > +    $\beta \in [0, 1]$ and $\delta \in \reals$,
> > +    then for $y \in [0,\infty)$,
> > +\[
> > +\distro{Wiener}(y|\alpha, \tau, \beta, \delta)
> > +=
> > +\frac{\alpha}{(y-\tau)^3/2}
> > +\exp \left(- \delta \alpha \beta - \frac{\delta^2(y-\tau)}{2}\right)
> > +\sum_{k = - \infty}^{\infty} (2k + \beta) \varphi \left(\frac{2k + \alpha \beta}{\sqrt{y - \tau}}\right)
> > +\!.
> > +\]
> > +
> > +where $\varphi(x)$ denotes the standard
>
> I just used the way it were in the article of Blurton2012, but you are
> right $\phi$ is more common.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stan-dev/stan/pull/1578/files#r36047123>.
>